module2/src/com/epam/arraysofarrays/MatrixWorkerOrderN.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.arraysofarrays;
+
+public class MatrixWorkerOrderN {
+    //task 4
+    public static  int[][] createSquareMatrixOrderPositiveN (int positiveN) {
+        int [][] matrix = new int[positiveN][positiveN];
+        for( int i = 0; i < positiveN; i++) {
+            for( int j = 0; j < positiveN; j++) {
+                if (i % 2 ==0) {
+                    matrix[i][j] = j + 1;
+                } else {
+                    matrix[i][j] = positiveN - j;
+                }
+            }
```

Comment:
Let's say here what is i % 2 == 0. In addition you don't need to check i % 2 == 0 every number, just every string or column
```suggestion
            boolean isOddRow = i % 2 == 0;
                if (isOddRow) {
                    for (int j = 0; j < orderOfMatrix; j++) {
                        matrix[i][j] = j + 1;
                    }
                } else {
                    for (int j = 0; j < orderOfMatrix; j++) {
                        matrix[i][j] = orderOfMatrix - j;
                    }
                }
```